plugins/scaffolder-actions/scaffolder-utils/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@roadiehq/scaffolder-utils",
```

Comment:
Can we call this:

@roadiehq/scaffolder-backend-module-utils

to fit with the pattern we already have for the other scaffolder package:

```@roadiehq/scaffolder-backend-module-http-request```

Reply:
done

Reply:
awsesome